gdnative-core/src/core_types/variant.rs

```diff
@@ -233,70 +233,112 @@ type Bool = bool;
 
 impl Variant {
     variant_constructors!(
-        /// Creates a `Variant` wrapping a `Vector2`.
+        /// Creates a `Variant` wrapping a [`Vector2`].
+        ///
+        /// [`Vector2`]: ../type.Vector2.html
```

Comment:
The function signatures that take the type aliases show the types from the `euclid` crate. Adding links to the type aliases here as I didn't find a way to use our type aliases instead of the `euclid` types.